infrequent-env

```diff
@@ -17,7 +17,8 @@ export JH_HOSTNAME=`echo ${JH_HOSTNAME} | sed -e's/tike/timeseries/g'`
 # Select "conda" tool used to install conda package and environments: conda or mamba
 #   mamba is largely a drop-in replacement for mamba with improved dependency resolution,
 #   parallel downloads, and C++ implementation.
-export CONDA_VER=mamba
+#export CONDA_VER=mamba
+export CONDA_VER=conda
```

Comment:
Not sure conda vs. mamba matters really,  but was this change intentional or accidental?  If accidental,  fine.  If intentional,  why?

Reply:
I was having dependency issues while building the images with mamba, swapped the conda, and seemed to work fine. 
My understanding is that it doesn't really matter which one we choose as far as it works.

Reply:
I think they're very similar,  but may actually have different package repos,  conda-forge vs...  something else.   I think docker-stacks is now using mamba which is one consideration...  but I think our switch to mamba was pretty arbitrary so there's no huge reason not to switch back.

Reply:
Swapped back to mamba

Reply:
Since mamba was throwing errors, swapped back to conda. Will investigate the issue.

Reply:
Manuel switched back to conda and was able to build on a CI-node.  jwebbinar failed for an unrelated reason so we're pressing ahead with the merge and transition to codecommit before further debug.